Dockerfile-clojure

```diff
@@ -1,7 +1,6 @@
 FROM clojure:alpine
 
-RUN apk add --update git perl \
-    && rm -rf /var/cache/apk
+RUN apk add --no-cache git perl tini
```

Comment:
(oh, and using `--no-cache` rather than the `--update` + delete the dir afterwards dance is better, and makes apk behave better)